help/README.md

```diff
@@ -0,0 +1,179 @@
+# Welcome to the New Help site
+
+The content of the Expensify Help is mainly composed of HTML and Markdown files (also referred as `.md` files in the documentation). We use [Jekyll](https://jekyllrb.com/) to generate a static site from those HTML and Markdown files.
+
+## HTML and Markdown basics
+
+We use the [heading markdown tags](https://www.markdownguide.org/cheat-sheet/) to create hierarchy in the article content (you can open this file in raw format and you will notice the `#` characters).
+
+In both HTML and Markdown files we use the [comment block](https://www.w3schools.com/html/html_comments.asp) to leave a note to the person who is editing the file. The content of the comment won't be rendered in the site.
+
+i.e.
+```
+<!-- This is a comment and won't be displayed in the page -->
+```
+
+# Test locally
+
+## Pre-requisites
+
+- Pull the [App repo](https://github.com/Expensify/App) in your computer.
+    - You can do it from from a Terminal command line by running `git clone https://github.com/Expensify/App.git`
+- Install jekyll in your computer by running `gem install bundler jekyll` in the command line.
+
+## Steps
+
+1. Open the Terminal command line in macOS.
+2. Navigate to the `help` folder in the command line.
+3. Run `jekyll serve`.
+4. Open localhost:4000 in the browser to see the site running locally.
+5. Modify a `.html` or `.md` file and refresh the page in the browser.
+
+# How the project is structured
+
+The [help](https://github.com/Expensify/App/tree/e6cea38ef22f83ae4730a34a9281d8962388dff8/help) folder will contain the following main folders:
+- *_layouts*: it will have at the moment only one file ([default.html](https://github.com/Expensify/App/blob/e6cea38ef22f83ae4730a34a9281d8962388dff8/help/_layouts/default.html)) which will render the common HTML elements and the assets on every page.
+- *_includes*: it will contain HTML content that can be reused on different pages. It will have the files home-navigation-tree.html, request-money-navigation-tree.html and  send-money-navigation-tree.html that will be rendered in the LHN.
```

Comment:
Ah afaik this is the only way to include HTML fragments in the pages https://jekyllrb.com/docs/includes/ otherwise Jekyll will treat it as another sub-page, but maybe I'm missing something related to including them inline. 

Meanwhile, I'm going to commit the suggestion

Reply:
> Ah afaik this is the only way to include HTML fragments in the pages [jekyllrb.com/docs/includes](https://jekyllrb.com/docs/includes/) otherwise Jekyll will treat it as another sub-page

Hmmm, well what I'm saying is that these HTML fragments aren't reused. AFAICT they're only used in default template like so:

```liquid
{% when "homepage" %}
    {% include home-navigation-tree.html %}
```

So what I'm wondering is could we just do this?

```liquid
{% when "homepage" %}
    <ul class="main-navigation-tree">
        <!-- LHN navigation html goes here -->
    </ul>
```

Reply:
Still NAB, mostly a curiosity to see if we can simplify things a bit so people don't have to research/understand jekyll includes